srcpkgs/libjxl/template

```diff
@@ -0,0 +1,76 @@
+# Template file for 'libjxl'
+pkgname=libjxl
+version=0.7.0
+revision=1
+# Build fails for armv7, see https://github.com/libjxl/libjxl/issues/1748
+archs="~armv7l"
+build_style=cmake
+configure_args="-DJPEGXL_ENABLE_BENCHMARK=OFF -DJPEGXL_ENABLE_EXAMPLES=OFF
+ -DJPEGXL_ENABLE_SJPEG=OFF -DJPEGXL_ENABLE_PLUGINS=ON"
+hostmakedepends="tar pkg-config asciidoc"
+makedepends="brotli-devel highway libpng-devel giflib-devel libjpeg-turbo-devel
+ libopenexr-devel libwebp-devel gdk-pixbuf-devel gimp-devel gtest-devel
+ xdg-utils"
+short_desc="JPEG XL image format reference implementation"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
+license="BSD-3-Clause, custom:Patent grant"
+homepage="https://jpeg.org/jpegxl/"
+_testdata_hash=d81acaca8e037d9514f3211045c77e7467bcb814
+_skcms_hash=64374756e03700d649f897dbd98c95e78c30c7da
+distfiles="https://github.com/libjxl/libjxl/archive/v${version}.tar.gz
+ https://github.com/libjxl/testdata/archive/${_testdata_hash}.tar.gz>testdata-${_testdata_hash}.tar.gz
+ https://skia.googlesource.com/skcms/+archive/${_skcms_hash}.tar.gz>skcms-${_skcms_hash}.tar.gz"
+checksum="3114bba1fabb36f6f4adc2632717209aa6f84077bc4e93b420e0d63fa0455c5e
+ 1a4d4be37f9a4dd69134832665fda8a772eeb95cbcdaa79d9b1850c23ae50521
+ @a69230c7b6f03a178c93abc6edc832c040d6e198340193b27c3d04afecf8f617"
+skip_extraction="testdata-${_testdata_hash}.tar.gz skcms-${_skcms_hash}.tar.gz"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	LIBS="-latomic"
+fi
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" -DBUILD_TESTING=OFF"
+fi
+
+post_extract() {
+	cd ${XBPS_SRCDISTDIR}/libjxl-${version}
+	tar -xf testdata-${_testdata_hash}.tar.gz --strip-components=1 -C ${wrksrc}/testdata
+	tar -xf skcms-${_skcms_hash}.tar.gz -C ${wrksrc}/third_party/skcms
+}
+
+post_install() {
+	vlicense LICENSE
+	vlicense PATENTS
+}
+
+libjxl-devel_package() {
+	short_desc+=" - development files"
+	depends="libjxl>=${version}_${revision} highway brotli-devel"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/lib/pkgconfig
+	}
+}
+
+libjxl-tools_package() {
+	short_desc+=" - tools"
+	pkg_install() {
+		vmove usr/bin
+		vmove usr/share/man
```

Comment:
```suggestion
		vmove usr/share/man/man1
```
if, e.g., they add man pages in section 3, we don't want them in this package

Reply:
Done.